__tests__/app.test.js

```diff
@@ -26,37 +26,89 @@ describe("GET /api/topics", () => {
     .get("/api/topics")
     .expect(200)
     .then(({ body: { topics } }) => {
-        const topicsEndpoint = endpointsJson["GET /api/topics"].exampleResponse.topics;
-
-        expect(Array.isArray(topics)).toBe(true);
-
         expect(topics.length).toBeGreaterThan(0);
 
         topics.forEach((topic) => {
           expect(topic).toHaveProperty("slug");
           expect(topic).toHaveProperty("description");
         })
+    })
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of all article objects", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({ body: { articles } }) => {
+        expect(articles.length).toBeGreaterThan(0);
 
-        expect(Object.keys(topics[0])).toStrictEqual(Object.keys(topicsEndpoint[0]));
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("body");
```

Comment:
we actually don't want the array of articles to have a body - so we should check it doesn't!